homework_4/numpy_challenge.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+if __name__ == "__main__":
+   array1_tarrantion_movies = np.array(['Reservoir Dogs ', 'Pulp Fiction', 
+'Jackie Brown', 'Kill Bill: Vol. 1', 'Kill Bill: Vol. 2', 'Death Proof', 
+'Inglourious Basterds',  'Django Unchained', 'The Hateful Eight', 'Once 
+Upon a Time in Hollywoo'])
```

Comment:
Скрипт не запускается, так как есть SyntaxError, обычные строки нельзя просто так переносить. Запускай код для проверки перед тем как сдавать
```suggestion
   array1_tarrantion_movies = np.array(['Reservoir Dogs ', 'Pulp Fiction', 
'Jackie Brown', 'Kill Bill: Vol. 1', 'Kill Bill: Vol. 2', 'Death Proof', 
'Inglourious Basterds',  'Django Unchained', 'The Hateful Eight', 'Once Upon a Time in Hollywoo'])
```